src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -302,7 +302,18 @@ public static TestPackage MakeTestPackage(ConsoleOptions options)
             if (options.Verbose)
                 package.AddSetting("Verbose", true);
 
+            if (options.DebugTests)
+            {
+                package.AddSetting(PackageSettings.DebugTests, true);
+
+                if (options.NumWorkers < 0)
+                    package.AddSetting(PackageSettings.NumberOfTestWorkers, 0);
```

Comment:
I thought that if the number of workers was 0, then it would use the default of `max(2, # processors)`. Is this not the case? I see the default is -1, so does 0 imply one imply everything is run on the main worker thread?